### PR TITLE
CLEANUP: Organize online_bagger

### PR DESCRIPTION
**Pull request by kingkevlar**
I organized online_bagger to match the existing file structure in mil_common and made the necessary (one line) change to the actual file (online_bagger.py) required from moving the .srv file from mil_tools to mil_msgs (with the rest of the msgs and srvs).

### CODE REVIEW

File: utils/mil_tools/mil_ros_tools/online_bagger/config/online_bagger_example.yaml [hunk 1]
```diff
@@ -1,15 +0,0 @@
-# list an arbitrary set of topics to  subscribe to and stream:
```
Comment (mattlangford) [inline utils/mil_tools/mil_ros_tools/online_bagger/config/online_bagger_example.yaml]: I think this is useful to keep as an example in an example launch.
Idea:
Move `online_bagger.py` into `utils/mil_tools/nodes`, then making a `utils/mil_tools/launch` where you put the launch file with the config in there as params in the launch.

**Review by mattlangford — APPROVED** [review]
A suggestion for organization, take it or leave it

File: utils/mil_tools/mil_ros_tools/online_bagger/config/online_bagger_example.yaml (hunk 1 above)
Comment (kingkevlar) [reply utils/mil_tools/mil_ros_tools/online_bagger/config/online_bagger_example.yaml]: Give me two more lines

**Review by sentree — CHANGES_REQUESTED** [review]
This will not work as it is. In order to build the message in the mil_msgs package, you need to add `BaggerCommands.srv` to the `add_service_files` block in `utils/mil_msgs/CMakeLists.txt`.

**Review by kev-the-dev — CHANGES_REQUESTED** [review]
The example launch file now refers to a config file that isn't there. Should change before merging

**Comment by sentree** [comment]
Is there a reason we're keeping the sample launch file and not the sample configuration file? Realistically, I think both of these should be in a readme with some good documentation.

    @kingkevlar --verbose

**Review by kev-the-dev — APPROVED** [review]

**Comment by kingkevlar** [comment]
@whispercoros It should be there now (both the config and launch file)

**Review by sentree — APPROVED** [review]

**Pull request merged**